start-servant.cabal

```diff
@@ -19,7 +19,7 @@ common lang-settings
 common common-dependencies
   build-depends:
       -- base (TODO: replace with base-noprelude and use Prelude from Protolude (as mixin))
```

Comment:
I guess you can remove the comment now. I think I'd like to state in the README a few things about how the code base is organized, e.g. that it uses Protolude.